docs/network-flow-visibility.md

```diff
@@ -142,26 +163,23 @@ the flow. All the IEs used by the Antrea Flow Exporter are listed below:
 
 #### Types of Flows and Associated Information
 
-Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Node,
-Node-to-Pod, Node-to-Node and Pod-to-Service network flows along with the associated
-statistics such as data throughput (bits per second), packet throughput (packets
-per second), cumulative byte count, cumulative packet count etc. Pod-To-Service
-flow visibility is supported only [when Antrea Proxy enabled](feature-gates.md).
+Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Service
+and Pod-to-External network flows along with the associated statistics such as data
+throughput (bits per second), packet throughput (packets per second), cumulative byte
+count and cumulative packet count. Pod-To-Service flow visibility is supported
+only [when Antrea Proxy enabled](feature-gates.md). In the future, we will extend
+the support to External-To-Service flows.
 
 Kubernetes information such as Node name, Pod name, Pod Namespace, Service name,
 NetworkPolicy name and NetworkPolicy Namespace, is added to the flow records. For
-flow records that are exported from any given Antrea Agent, we only provide the
-information of Kubernetes entities that are local to the Antrea Agent. In the future,
-we plan to extend this feature to provide information about remote Kubernetes entities
-such as remote Node name, remote Pod name etc.
-
-Please note that in the case of inter-Node flows, we are exporting only one copy
-of the flow record from the source Node, where the flow originates from, and ignore
-the flow record from the destination Node, where the destination Pod resides. Due
-to this we miss key information such as destination Pod info, ingress NetworkPolicy
-info, stats from the destination Node, etc. In the future, this behavior will be
-changed when we add support for correlating the different flow records (from source
-and destination Nodes) that belong to the same flow.
+flow records that are exported from any given Antrea Agent, the Flow Exporter only
+provides the information of Kubernetes entities that are local to the Antrea Agent.
+In other words, flow records are only complete for intra-Node flows, but incomplete
+for inter-Node flows.
```

Comment:
Maybe add an extra sentence at the end here: `It is the responsibility of the (Flow Aggregator)[#flow-aggregator]` to correlate flows from source and destination Nodes and produce complete flow records.`